project/Dependencies.scala

```diff
@@ -72,7 +65,7 @@ object Dependencies {
   private val guavaV = "31.0.1-jre"
   private val heterodonV = "1.0.0-beta3"
   private val hsqldbV = "2.6.0"
-  private val http4sV = "0.21.7" // scala-steward:off (CROM-6678)
+  private val http4sV = "0.21.31"
```

Comment:
Just putting this comment in for future context-- this release is EOL. We need to upgrade further for cats3. https://http4s.org/versions/

Reply:
Hmm... can we go to 0.22.8 now? At least that one isn't EOL.

Reply:
Beyond the scope of this ticket but coming soon as part of the cats3 upgrade if not sooner.